src/templates/Vue InstantSearch 1/package.json

```diff
@@ -3,35 +3,33 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "cross-env NODE_ENV=development webpack-dev-server --port 3000 --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
-    "lint": "eslint --ext .js,.vue .",
+    "start": "vue-cli-service serve --port 3000",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
     "lint:fix": "npm run lint -- --fix"
   },
   "dependencies": {
-    "vue": "2.6.2",
+    "core-js": "2.6.5",
+    "vue": "2.6.10",
     "vue-instantsearch": "{{libraryVersion}}"
   },
   "devDependencies": {
-    "babel-core": "6.26.3",
+    "@vue/cli-plugin-babel": "3.6.0",
+    "@vue/cli-plugin-eslint": "3.6.0",
+    "@vue/cli-service": "3.6.0",
     "babel-eslint": "10.0.1",
-    "babel-loader": "7.1.5",
-    "babel-preset-env": "1.7.0",
-    "babel-preset-stage-3": "6.24.1",
-    "cross-env": "5.2.0",
-    "css-loader": "0.28.11",
     "eslint": "5.7.0",
     "eslint-config-algolia": "13.2.3",
```

Comment:
I wonder if we even need this, what if we just use the defaults?

Reply:
We use it everywhere inside our project, I think it make sense to keep it.